en/csql.rst

```diff
@@ -1412,11 +1412,11 @@ The **format** or **fmt** option allows you to edit after formatting the SQL sta
 
 **Specifying the formatter (;FOrmatter_cmd)**
 
-This command specifies the formatter to be used with **;EDIT** session command. The user can set it up and use it as shown in the example, and OS environment variable(FORMATTER) can be set up in advance for use. ::
+This command specifies the formatter to be used with **;EDIT** session command. The user can set it up and use it as shown in the example, and OS environment variable(CUBRID_CSQL_FORMATTER) can be set up in advance for use. ::
```

Comment:
This command specifies the formatter to be used with **;EDIT** session command. The user can set it up and use it as shown in the example, and OS environment variable(either CUBRID_CSQL_FORMATTER or EDITOR) can be set up in advance for use. ::